ros2component/ros2component/api/__init__.py

```diff
@@ -82,7 +81,10 @@ def get_container_components_info(*, node, remote_container_node_name):
         '{}/_container/list_nodes'.format(remote_container_node_name)
     )
     try:
-        list_nodes_client.wait_for_service()
+        if not list_nodes_client.wait_for_service(timeout_sec=5.0):
+            raise RuntimeError(
+                f"No list_nodes service found for '{remote_container_node_name}' container"
```

Comment:
The service name should be quoted for better readability.

Same below.

Reply:
See 5a28d9da97d673a795a3e911cd6e843765399065.